recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ about:
   home: https://github.com/graingert/pkgutil-resolve-name
```

Comment:
Since this is a backport of a python module, there should be a skip for python higher than 3.8.